addon/components/date-range-picker.js

```diff
@@ -80,6 +84,9 @@ export default Ember.Component.extend({
     this.setupPicker();
   },
 
+  isInvalidDate: function(){},
+  isCustomDate: function(){},
```

Comment:
Does this means that you should pass a function to the component if you wants to change the behavior?